### PR TITLE
Add .sum extension

### PR DESCRIPTION
**Pull request by csh519**
There are some FP cases from .sum files.
.sum file is checksum file of golang, so it can be excluded from scanning.

### CODE REVIEW

File: credsweeper/secret/config.json [hunk 1]
```diff
@@ -145,4 +145,4 @@
         "ml_validation",
         "ml_probability"
     ]
-}
+}
```
Comment (babenek) [inline credsweeper/secret/config.json:148]: too many extra changes :(

**Review by babenek — APPROVED** [review]
ok. ```.sum ``` should be skipped

File: credsweeper/secret/config.json (hunk 1 above)
Comment (babenek) [inline credsweeper/secret/config.json:148]:
```suggestion
}

```

**Review by babenek — CHANGES_REQUESTED** [review]
please, fix line ending

**Review by babenek — APPROVED** [review]
LGTM

**Review by kmnls — APPROVED** [review]

**Comment by babenek** [comment]
@csh519 , sorry, i wanted merge the changes from develop to resolve conflicts in next PRs.
Will merge from your fork next time

**Pull request merged**